torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -85,6 +86,18 @@ auto CopySlices::apply(variable_list&& inputs) -> variable_list {
     grad_slice = result.as_strided(view.sizes(), view.strides(), offset);
   }
 
+  // Adding the missing nodes to the current graph's `exec_info`.
+  // This is a workaround because the current `GraphTask::init_to_execute`
+  // does not traverse into CopySlices node.
+  const auto exec_info = get_current_graph_task_exec_info();
+  if (exec_info && !exec_info->empty()) {
+    for (const auto& next : fn->next_edges()) {
+      if (next.is_valid()) {
+        add_node_to_current_graph_task_exec_info(next.function.get());
+      }
+    }
+  }
```

Comment:
Is there a test that fails if we don't have this?

Reply:
yes, the following test will fail if we don't have this
```
FAILED test_autograd.py::TestAutograd::test_inplace_on_view_backward - RuntimeError: res[i].defined() INTERNAL ASSERT FAILED at "/home/jinzex/dev/pytorch1/torch/csrc/autograd/functions/tensor.cpp":109, pleas...
FAILED test_autograd.py::TestAutograd::test_lobpcg - RuntimeError: res[i].defined() INTERNAL ASSERT FAILED at "/home/jinzex/dev/pytorch1/torch/csrc/autograd/functions/tensor.cpp":109, please report a bug to ...
FAILED test_autograd.py::TestAutogradDeviceTypeCPU::test_inplace_on_view_gradcheck_cpu - RuntimeError: res[i].defined() INTERNAL ASSERT FAILED at "/home/jinzex/dev/pytorch1/torch/csrc/autograd/functions/tens...
FAILED test_autograd.py::TestAutogradDeviceTypeCUDA::test_inplace_on_view_gradcheck_cuda - RuntimeError: res[i].defined() INTERNAL ASSERT FAILED at "/home/jinzex/dev/pytorch1/torch/csrc/autograd/functions/te...

```